packages/flutter/lib/src/widgets/form.dart

```diff
@@ -22,6 +22,7 @@ class Form extends StatefulWidget {
   Form({
     Key key,
     @required this.child,
+    this.autovalidate: false,
```

Comment:
I thought so too, but grepping through the codebase, I found `autofocus` and `autorefresh`. There are no `autoFoos` anywhere. WDYT?